File: app/src/main/java/com/seletiva/santander/investment/Models/CellType.java
package com.seletiva.santander.investment.Models;

public enum CellType {
    field(1),
    text(2),
    image(3),
    checkbox(4),
    send(5);

    private int type;

    CellType(int type) {
        this.type = type;
    }

    public int getType() {
        return type;
    }
}

File: STDAndroidTeste/app/src/main/java/br/com/cesarsicas/stdandroidteste/constants/CellType.kt
package br.com.cesarsicas.stdandroidteste.constants
import br.com.cesarsicas.stdandroidteste.adapters.TypeAdapter
import com.google.gson.annotations.JsonAdapter

/**
 * Created by julio on 4/19/18.
 */
@JsonAdapter(TypeAdapter::class)
enum class CellType(var value:Int) {
    field(1),
    text(2),
    image(3),
    checkbox(4),
    send(5);

    companion object {
        fun from(findValue: Int): CellType = CellType.values().first { it.value == findValue }
    }
}